### DISCUSSION TITLE
[Skia][WPF] The local function 'EnqueueNative' is declared but never used

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#7794] [Q&A] [answered]
https://github.com/unoplatform/uno/blob/acfac1b01148869993e3fcc7cb3a6755b1ee9ac1/src/Uno.UI.Runtime.Skia.Wpf/WpfHost.cs#L107-L125

Can be removed?

**Comment by jeromelaban** [comment, marked as answer]
That's correct, it was incorrectly left there as a result of the migration to `DispatcherQueue`. It should be removed. You can make a PR if you want :)
